src/style.css

```diff
@@ -0,0 +1,29 @@
+* {
+  margin: 0;
+}
+
+body {
+  background-color: black;
+  margin: 20px;
+}
+
+div {
+  box-sizing: border-box;
+  display: block;
+  border: 1px solid #e65451;
+  padding: 10px;
+}
```

Comment:
Try to avoid stylizing tags. Always use classes for this. Tags stylizing is OK when you reset default browser styles, like margin: 0 for BODY tag, which has 8px margins. Setting width and height for all DIV tags is bad practice